src/components/Boundary.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable react/prop-types */
+
+import React, { Component } from 'react';
+import * as Sentry from '@sentry/browser';
+
+export default class Boundary extends Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+			eventId: null,
+			hasError: false,
+		};
+	}
```

Comment:
```suggestion
		state = {
			eventId: null,
			hasError: false,
		}
```
`state` можно объявлять как свойство без переопределения конструктора